$MirOS: ports/www/dillo/v2/patches/patch-dpi_cookies_c,v 1.4 2010/02/21 20:08:42 bsiegert Exp $
--- dpi/cookies.c.orig	Sat Jun 27 14:42:12 2009
+++ dpi/cookies.c	Mon Aug 24 16:24:38 2009
@@ -244,13 +244,13 @@ static void Cookies_init()
    }
 
    /* Get a stream for the cookies file */
-   filename = dStrconcat(dGethomedir(), "/.dillo/cookies.txt", NULL);
+   filename = dStrconcat(dGethomedir(), "/.etc/dillo/cookies.txt", NULL);
    file_stream = Cookies_fopen(filename, "r+", cookies_txt_header_str);
 
    dFree(filename);
 
    if (!file_stream) {
-      MSG("ERROR: Can't open ~/.dillo/cookies.txt, disabling cookies\n");
+      MSG("ERROR: Can't open ~/.etc/dillo/cookies.txt, disabling cookies\n");
       return;
    }
 
@@ -341,9 +341,9 @@ static void Cookies_init()
       }
    }
 
-   filename = dStrconcat(dGethomedir(), "/.dillo/cookies", NULL);
+   filename = dStrconcat(dGethomedir(), "/.etc/dillo/cookies", NULL);
    if ((old_cookies_file_stream = fopen(filename, "r")) != NULL) {
-      MSG("WARNING: Reading old cookies file ~/.dillo/cookies too\n");
+      MSG("WARNING: Reading old cookies file ~/.etc/dillo/cookies too\n");
 
       /* Get all lines in the file */
       while (!feof(old_cookies_file_stream)) {
@@ -577,8 +577,8 @@ static time_t Cookies_create_timestamp(c
                   (minutes * 60) +
                   seconds);
 
-   MSG("Expires in %ld seconds, at %s",
-       (long)ret - time(NULL), ctime(&ret));
+   MSG("Expires in %lld seconds, at %s",
+       (long long)ret - time(NULL), ctime(&ret));
 
    return ret;
 }
@@ -1237,7 +1237,7 @@ static int Cookie_control_init(void)
    bool_t enabled = FALSE;
 
    /* Get a file pointer */
-   filename = dStrconcat(dGethomedir(), "/.dillo/cookiesrc", NULL);
+   filename = dStrconcat(dGethomedir(), "/.etc/dillo/cookiesrc", NULL);
    stream = Cookies_fopen(filename, "r", "DEFAULT DENY\n");
    dFree(filename);
 
